mirdata/datasets/acousticbrainz_genre.py

```diff
@@ -383,8 +393,7 @@ def load_extractor(path):
 
 @core.docstring_inherit(core.Dataset)
 class Dataset(core.Dataset):
-    """The acousticbrainz genre dataset
-    """
+    """The acousticbrainz genre dataset"""
```

Comment:
@magdalenafuentes (thanks to @drubinstein for the fix) the change in formatting you made to the docstring - 
"""blablabla""""
vs how it was
"""blablabla
"""
is what's causing the formatting problem in the docs build.

Reply:
one more note from @drubinstein - 
black will reformat
"""blablabla
"""
to one line, but it wont reformat:
"""
blablabla
"""
which will build properly too.

Reply:
Ooops, good to know, so it should be either """blabla""" or 
""""blabla
""""?

Reply:
Not exactly.
```
"""A"""
```
Will cause the problem

```
"""A
"""
```
or 
```
"""
A
"""
```

are acceptable solutions. However, black will format solution 1 to look like the version that causes the problem therefore solution 2 is recommended.

Reply:
mmm ok, will change things to be that format then

Reply:
btw thanks @drubinstein for the hint!